frontend/src/components/Pagination.js

```diff
@@ -6,28 +6,25 @@ import { ButtonGroup, Button } from "react-bootstrap";
 const Pagination = (props) => {
   let renderPaginationBox = (content, pageIndex) => (
     <Button
-      key={`pagination-box-${pageIndex}`}
+      key={`pagination-box-${content}`}
```

Comment:
What does changing this to content do?

Reply:
Sorry for the late reply. This is because pageIndex can be duplicated, for example, button "First" and button "Page 1" both have pageIndex 0, which leads to duplicated key.